### PR TITLE
Added more customizability to the Discord Rich Presence Plugin on Cider (Buttons)

### PR DESCRIPTION
**Pull request by Monochromish**
Now, People can choose whether to hide buttons on their rich presence (Listen on Cider & View on Apple Music)

### CODE REVIEW

File: src/i18n/en_US.json [hunk 1]
```diff
@@ -1,4 +1,5 @@
 {
+<<<<<<< Updated upstream
```
Comment (coredev-uk) [inline src/i18n/en_US.json:2]: Fix ur merge comments

File: src/i18n/en_US.json [hunk 2]
```diff
@@ -426,4 +427,434 @@
   "share.platform.songLink": "Copy with song.link",
   "share.platform.clipboard": "Copy Link",
   "about.thanks": "Major thanks to the Cider Collective Team and all of our contributors."
+=======
```
Comment (coredev-uk) [inline src/i18n/en_US.json:430]: here again

File: src/i18n/en_US.json [hunk 3]
```diff
@@ -426,4 +427,434 @@
...
+	"share.platform.facebook": "Facebook",
+	"share.platform.reddit": "Reddit",
+	"share.platform.telegram": "Telegram",
+	"share.platform.whatsapp": "WhatsApp",
+	"share.platform.messenger": "Messenger",
+	"share.platform.email": "Email",
+	"share.platform.songLink": "Copy with song.link",
+	"share.platform.clipboard": "Copy Link",
+	"about.thanks": "Major thanks to the Cider Collective Team and all of our contributors."
+>>>>>>> Stashed changes
```
Comment (coredev-uk) [inline src/i18n/en_US.json:859]: here

File: src/i18n/source/en_US.json [hunk 4]
```diff
@@ -1,4 +1,5 @@
 {
+<<<<<<< Updated upstream
```
Comment (coredev-uk) [inline src/i18n/source/en_US.json:2]: here

File: src/i18n/source/en_US.json [hunk 5]
```diff
@@ -426,4 +427,434 @@
   "share.platform.songLink": "Copy with song.link",
   "share.platform.clipboard": "Copy Link",
   "about.thanks": "Major thanks to the Cider Collective Team and all of our contributors."
+=======
```
Comment (coredev-uk) [inline src/i18n/source/en_US.json:430]: here

File: src/main/plugins/discordrpc.ts [hunk 6]
```diff
@@ -125,12 +169,97 @@ export default class DiscordRichPresence {
             delete activity.startTime
             delete activity.endTime
         }
+=======
```
Comment (coredev-uk) [inline src/main/plugins/discordrpc.ts:172]: c'mon

File: src/main/plugins/discordrpc.ts [hunk 7]
```diff
@@ -1,9 +1,53 @@
...
+		smallImageKey: '',
+		smallImageText: '',
+		instance: false
+	};
+
+	/*******************************************************************************************
+	 * Private Methods
+	 * ****************************************************************************************/
+
+<<<<<<< Updated upstream
```
Comment (coredev-uk) [inline src/main/plugins/discordrpc.ts:50]: check

**Review by coredev-uk — CHANGES_REQUESTED** [review]
A fuck ton of merge comments, I've only commented on a few of them but there's more. I suggest you restore your i18n files to source.

**Comment by rowbawts** [comment]
I'd love to see this as I believe it will contribute to a cleaner look for the app's integration.

**Comment by Monochromish** [comment]
Sorry for the inconvenience @coredev-uk; I've reviewed the comments.

**Comment by maikirakiwi** [comment]
Check branch conflict

**Review by coredev-uk — CHANGES_REQUESTED** [review]
I suggest you remove the formatting that you have done on various files, it's hard to track what code you are contributing when there are large diffs. Sorry.

**Comment by coredev-uk** [comment]
Requested close, pending new PR with the feature from the author.

**Pull request closed without merging**